storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst

```diff
@@ -0,0 +1,39 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.13/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+Improvements and New Features
+================================================================================
+
+- :jirabug:`1548`: |Percona Xtrabackup| enables updating the
+  ib_buffer_pool file with the latest pages present in the buffer pool by
+  setting the ``innodb_buffer_pool_dump_now`` variable to `ON` .
+- :jirabug:`1609`: The ``--version_check`` option can be disabled at build time
+
+Bus Fixed
+================================================================================
+
+- :jirabug:`1536`: |Percona Xtrabackup| 2.4.11, MySQL 5.7.21-21-log Percona Server partial backup and errors in MySQL log
+- :jirabug:`1569`: The ``--history`` option did not work when autocommit was disabled
+- :jirabug:`1570`: |Percona XtraBackup| was affected by issue `PS-4519 <https://jira.percona.com/browse/PS-4519>`_ (Percona Server for MySQL could not be compiled on CentOS 6 with gcc 6.3.1)
+- :jirabug:`1648`: ``xtrabackup`` could fail to open tablespace with the error "Encryption information in datafile: <NAME.ibd> can't be decrypted"
+- :jirabug:`1668`: When the ``--throttle`` option was used, the applied value was different from the one specified by the user (off by one error)
+- :jirabug:`1672`: It was not allowed for MTS (multi-threaded slaves)  without GTID to be backed up with ``--safe-slave-backup``
+- :jirabug:`1679`: |Percona Xtrabackup| could crash when the ``ALTER TABLE … TRUNCATE PARTITION`` command was run during a backup without locking DDL
+- :jirabug:`1683`: ``xbcrypt`` could display an assertion failure and generated core if the required parameters are missing
+- :jirabug:`1691`: The "out of memory" error was reported when making a backup with ``--lock-ddl-per-table``
```

Comment:
can we combine 1691 and 1698 into one with description "Using --lock-ddl-per-table caused server to scan all records of partitioned tables which could lead to out of memory error"

Reply:
Updated as requested